.github/workflows/build_deploy_exm_fe.yml

```diff
@@ -0,0 +1,100 @@
+name: Build and Deploy EXM FE
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    name: Build Angular Application
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Download and install Maven
+      run: sudo apt-get update && sudo apt-get install -y maven
+
+    - name: Build Angular application using Maven
+      run: mvn clean install
+      working-directory: expenses-manager-frontend
+
+    - name: Upload build artifacts
```

Comment:
Nu vrem sa facem upload la artifact daca se face build, poti pune is aici acelasi if ca la build-ul de deploy